cypress/e2e/calendar/purge-old-calendars.cy.js

```diff
@@ -0,0 +1,91 @@
+import {
+  MultiColumnListCell, Modal, Pane, Button, Link, including, Select, Accordion, ListItem
+} from '../../../interactors';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+// set calendar start and end date to at least 3 months ago
+testCalendar.startDate = '2022-01-01';
+testCalendar.endDate = '2022-01-31';
+testCalendar.exceptions = [];
+
+
+
+describe('Purge old calendars with a selectable date', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+    });
+
+    cy.openCalendarSettings();
+  });
+
+  after(() => {
+    cy.deleteServicePoint(testServicePoint.id, true);
+  });
+
+
+  it('purges old calendar with a selectable date', () => {
```

Comment:
it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {